unfoldingWord/translationCore
# Give error when trying to import project from door43 with no internet access

#### This pull request addresses:
Addresses https://github.com/unfoldingWord-dev/translationCore/issues/1257



#### How to test this pull request:

Login to Door43 while online. 
Disconnect from the internet
Try and view your list of Door43 repos on the projects tab
You should see a new error message prompting you to check your internet connection.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1416)
